src/amo/components/CollectionList/styles.scss

```diff
@@ -0,0 +1,58 @@
+@import "~core/css/inc/mixins";
+@import "~photon-colors/photon-colors";
+@import "~ui/css/vars";
+
+.CollectionList {
+  padding: $padding-page;
+
+  @include respond-to(large) {
+    padding: $padding-page-l;
+  }
+
+  .Card-contents {
+    font-size: $font-size-m-smaller;
+  }
+}
+
+@include respond-to(large) {
+  .CollectionList-wrapper {
+    display: grid;
+    grid-auto-flow: column dense;
+    grid-gap: 0 $padding-page-l;
```

Comment:
On small screens, there is no padding/margin between the two cards:

![screen shot 2018-06-22 at 14 19 39](https://user-images.githubusercontent.com/217628/41776283-6f9b76a6-7627-11e8-9ae1-70b73b2350e4.png)